src/jquery.vide.js

```diff
@@ -331,7 +336,8 @@
       transform: 'translate(-' + position.x + ', -' + position.y + ')',
 
       // Disable visibility, while loading
-      visibility: 'hidden'
+      visibility: 'hidden',
+      opacity: 0
```

Comment:
Hidden hides everything except this "blur" effect that appears behind the system play button. It's likely a bug in iOS9, but this fixes it.